README.md

```diff
@@ -27,7 +27,7 @@ receivers:
 ...
 - name: cos-alerter
   webhook_configs:
-  - url: http://<cos-alerter-address>:8080/alive?clientid=<clientid>
+  - url: http://<cos-alerter-address>:8080/alive?clientid=<clientid>&key=<clientkey>
```

Comment:
Should we add a comment here in the the sample doc explaining what `clientid` and `clientkey` are for?
Not obvious to me at least.

Reply:
The description should go in the default config file, which is where you currently have it but maybe a few words on what each value should be used for would be good.